server/image_pull.go

```diff
@@ -92,6 +92,8 @@ func (s *Server) PullImage(ctx context.Context, req *pb.PullImageRequest) (resp
 				break
 			}
 			logrus.Debugf("image in store has different ID, re-pulling %s", img)
+		} else {
+			return nil, err
```

Comment:
nope sorry, this is not right, we shouldn't have this as otherwise, we won't try pulling the next image in the ResolveNames list. This is a best effort check to check if an image is on disk before even try pulling, we don't want to throw an error in this case

Reply:
maybe #1548 can be closed, did not find other swallow errors mentioned in this issue.